default.build

```diff
@@ -170,9 +170,12 @@
   <target name="build_windows10" description="Build Windows 10 UAP" depends="clean">
     <if test="${os == 'Win32NT'}">
       <if test="${file::exists('c:\Program Files (x86)\MSBuild\Microsoft\WindowsXaml\v14.0\Microsoft.Windows.UI.Xaml.CSharp.targets')}">
-        <exec program="Protobuild" commandline="-generate WindowsUAP" />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU"' />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU"' />
+        <exec program="Protobuild" commandline="-generate WindowsUniversal" />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
```

Comment:
> The TeamCity server does all the git work. The build agent on another machine gets sent the content of the repo for that one revision it should be building at that time. The .git folder never makes it to the build agents.

That's... an interesting way of doing things.  Does this imply it copies the entire repository over the network to each build agent every time?

I'll change this to be "if neither .git or .pkg exist, and there's more than one file/subfolder in the target directory, leave it alone" since that's a safer option anyway.

> So while I appreciate that this is a great feature for some of your users... please respect me when I say we don't need it in the MonoGame project. It is becoming a problem that we constantly have to fight for Protobuild not to force its new features into MonoGame.

This isn't about my users; I don't have users.  These are features that are useful for people using MonoGame.

I don't think I've seen anyone else raise issues with new functionality other than you; the other maintainers seem fine with it.  The context of raising the issue is always "I personally don't work with MonoGame like this", which is not necessarily the same as "everyone using MonoGame doesn't work like this".  I don't want to turn off useful functionality because one person isn't using it; that doesn't make sense.